modules/ROOT/pages/as2/as2-connector-upgrade-migrate-6-0.adoc

```diff
@@ -0,0 +1,90 @@
+= AS2 Connector 6.0.0 - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.x |6.x
+|===
+
+== Changes in This Release (v6.0)
+
+This release contains the following changes:
+
+* Digitally signed copies of outgoing and received messages and their corresponding receipts can be saved for Non-repudiation purposes as long as they are configured and enabled on AS2 connector operations and sources.
+
+* The new Mule source `Non repudiation listener` must be user-configured to persist the Non-Repudiation information provided by any of the AS2 Connector
+operations and sources via a user-defined persistence strategy.
+
+* Certain attribute object fields have been renamed and deprecated fields removed. These are outlined in the Upgrade Steps.
+
+== Requirements and Limitations
+
+You should be aware of some requirements and limitations before you upgrade to the latest version.
+
+=== Requirements
+
+Ensure that you meet the following requirements before you migrate.
+
+* Mule 4.3.0 or later
+
+* Java 8
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade to AS2 version 6.0.0:
+
+. Download the AS2 connector 6.0.0 from Exchange and place appropriate source/operation in the flow that is being upgraded.
+
+=== Changes for AS2 Listener and AS2 MDN Listener
+
+There are no changes to the way AS2 Listener and AS2 MDN Listener is used or configured.
+
+=== Changes for Send with Sync MDN and Send with Async MDN operations
+
+For `Send with Sync MDN` and `Send with Async MDN` operations:
+
+* An attribute object of type `com.mulesoft.connector.as2.api.AS2MdnAttributes` has been changed for v6.0.0. The field `com.mulesoft.connector.as2.api.AS2SendAttributes.as2MdnAttributes` is of type `com.mulesoft.connector.as2.api.AS2MdnAttributes`).
+
+* The MultiMap `dispositionContent` has been changed into an object called `DispositionNotification` which now has each individual field of the message `disposition-notification` part of the mdn. The `DispositionNotification` can be found in the `DispositionNotificationReport` object.
```

Comment:
The MultiMap `dispositionContent` has been converted into an object called `DispositionNotification` splits the information of the `disposition-notification` part of the mdn into parameters. The `DispositionNotification` can be found in the `DispositionNotificationReport` object.